src/plugins/vis_augmenter/public/utils/utils.ts

```diff
@@ -50,6 +54,15 @@ export const getAugmentVisSavedObjs = async (
   visId: string | undefined,
   loader: SavedAugmentVisLoader | undefined
 ): Promise<ISavedAugmentVis[]> => {
+  if (visId === undefined || loader === undefined) {
+    return [] as ISavedAugmentVis[];
+  }
```

Comment:
These can be removed. The logic using these args are wrapped in `try/catch`

Reply:
Nice catch. Forgot to clean this up after testing out the functionality and debugging.

Reply:
Thanks - you can resolve